test/scalability/runner/recorder/recorder.go

```diff
@@ -18,27 +18,138 @@ package recorder
 
 import (
 	"context"
+	"encoding/csv"
+	"os"
+	"strconv"
 	"sync/atomic"
 	"time"
 
 	apimeta "k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/yaml"
 
 	kueue "sigs.k8s.io/kueue/apis/kueue/v1beta1"
+	"sigs.k8s.io/kueue/test/scalability/runner/generator"
 )
 
+type event interface {
```

Comment:
Is this needed?

Reply:
just not to go `interface{}` on the events channel, same goes for it method.

Reply:
I see, I'm ok keeping the dedicated `event` interface, but it's function does not seem to be used. We can remove the declaration and implementations I suppose.

Reply:
Making `event` an empty interface will defeat the purpose of having it (restrict what types qualify as an `event`), if you insist we can just use `interface {}`.

Reply:
Having a dedicated interface sgtm as a place for some functions. I prefer to make it empty, but I'm ok with the dummy function. I was just thinking if I'm not missing some uses.

Reply:
It just occurred to me that "|" types within the interface might do it without the need of the dummy func.

Reply:
how would it look like?

Reply:
Unfortunately it's not working, we can go with different channels (for event type).

Reply:
ok

Reply:
done